pootle/apps/pootle_project/models.py

```diff
@@ -512,6 +525,13 @@ def _get_code(self, project):
 
 @receiver([post_delete, post_save])
 def invalidate_resources_cache(sender, instance, **kwargs):
+    if instance.__class__.__name__ == 'VirtualFolder':
+        cache.delete_many([
```

Comment:
This looks wrong, not sure if you tested it: `cache.delete_many()` receives a list of keys to be removed from the cache, but here you have a list comprehension which calls `cache.delete()` (!!!).

Since we already enforce *django-redis*, you can `cache.delete_pattern(make_method_key('Project', 'resources', '*'))`. But please test it.